Core/src/org/sleuthkit/autopsy/casemodule/CasePropertiesAction.java

```diff
@@ -61,32 +58,16 @@ public void performAction() {
             // create the popUp window for it
             String title = NbBundle.getMessage(this.getClass(), "CasePropertiesAction.window.title");
             final JFrame frame = new JFrame(title);
-            popUpWindow = new JDialog(frame, title, true); // to make the popUp Window to be modal
-
-            // get the information that needed
-            Case currentCase = Case.getCurrentCase();
-            String crDate = currentCase.getCreatedDate();
-            String caseDir = currentCase.getCaseDirectory();
-
-            // put the image paths information into hashmap
-            Map<Long, String> imgPaths = Case.getImagePaths(currentCase.getSleuthkitCase());
+            popUpWindow = new JDialog(frame, title, false); // to make the popUp Window to be modal
```

Comment:
Please make either make the comment match the code or delete the comment (it is not really needed).